pkgs/development/libraries/decaf/default.nix

```diff
@@ -0,0 +1,38 @@
+{ cmake
+, fetchFromGitLab
+, lib
+, python3
+, stdenv
+}:
+
+stdenv.mkDerivation rec {
+  pname = "decaf";
+  version = "linphone-4.4.1";
+
+  nativeBuildInputs = [ cmake ];
+  buildInputs = [
+    python3
+  ];
+
+  src = fetchFromGitLab {
+    domain = "gitlab.linphone.org";
+    group = "BC";
+    owner = "public/external";
+    repo = pname;
+    rev = "6e78a9beb24d1e3d7050dd52a65e4f88b101a1fc";
+    sha256 = "sha256-D2SzkinloL0Ya9p25YUsc+7lKvoTMUsdkKrkv/5AEeY=";
+  };
+
+  # Do not build static libraries
+  cmakeFlags = [ "-DENABLE_STATIC=NO" ];
+
+  NIX_CFLAGS_COMPILE = [ ];
```

Comment:
Can't you leave this line out?

Reply:
Well, it turns out I had to keep it because I hit some warnings (interpreted as errors) when rebasing